pydantic/types.py

```diff
@@ -1719,7 +1719,8 @@ def validate_brand(card_number: str) -> PaymentCardBrand:
     'eib': 2**60,
 }
 BYTE_SIZES.update({k.lower()[0]: v for k, v in BYTE_SIZES.items() if 'i' not in k})
-byte_string_re = re.compile(r'^\s*(\d*\.?\d+)\s*(\w+)?', re.IGNORECASE)
+byte_string_pattern = r'^\s*(\d*\.?\d+)\s*(\w+)?'
```

Comment:
Can these be private (name them with a leading underscore)?

Reply:
Possibly also they could be put on the class as a namespace instead of the module

Reply:
good catch! made the change.